2-mandatory/2-currency-code-lookup.js

```diff
@@ -18,7 +18,13 @@ const COUNTRY_CURRENCY_CODES = [
 ];
 
 function createLookup(countryCurrencyCodes) {
-  // write code here
+  const newObject = {};
```

Comment:
Think about a better name. What does this variable represent? Would `lookup` name work better?